<ui version="4.0" >
 <class>qdFindFiles</class>
 <widget class="QDialog" name="qdFindFiles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find files</string>
  </property>
  <widget class="QPushButton" name="qpbCancel" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>340</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Found files:</string>
   </property>
   <widget class="QTreeWidget" name="qtwSearch" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="rootIsDecorated" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Go to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Feed to panel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="qpbStart" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbStop" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>S&amp;top</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>150</y>
       <width>351</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title" >
      <string>Full text</string>
     </property>
     <widget class="QComboBox" name="comboBox_3" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>331</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Whole words</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Not containing text</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>140</x>
        <y>70</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Hex</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>140</x>
        <y>90</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Regular expression</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Case sensitive</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Search for text</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title" >
      <string>Files</string>
     </property>
     <widget class="QCheckBox" name="checkBox" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>100</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Search in selected directories/files</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>80</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Regular expression</string>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>23</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Search &amp;for</string>
      </property>
      <property name="buddy" >
       <cstring>qcbSearchFor</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>53</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Search &amp;in</string>
      </property>
      <property name="buddy" >
       <cstring>qcbSearchIn</cstring>
      </property>
     </widget>
     <widget class="QComboBox" name="qcbSearchFor" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>20</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="qcbSearchIn" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>50</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbDrives" >
      <property name="geometry" >
       <rect>
        <x>260</x>
        <y>50</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>&amp;Drives</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbBrowse" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>&amp;Browse</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Advanced</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title" >
      <string>Date/time</string>
     </property>
     <widget class="QCheckBox" name="checkBox_9" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>23</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Between</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>23</y>
        <width>18</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>and</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_10" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>53</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Not older than</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>80</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_2" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>230</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value" >
       <number>1</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_11" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>113</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>File size</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>110</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Load/save</string>
    </attribute>
    <widget class="QListWidget" name="listWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4" >
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qcbSearchFor</tabstop>
  <tabstop>qpbDrives</tabstop>
  <tabstop>qcbSearchIn</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>qpbStart</tabstop>
  <tabstop>qpbCancel</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>comboBox_4</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>comboBox_5</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_9</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>qdFindFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>203</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
